packages/backend/src/nest/storage/userProfile/userProfile.store.spec.ts

```diff
@@ -164,37 +164,37 @@ describe('UserProfileStore/validateUserProfile', () => {
     const pngByteArray = new Uint8Array(pngArray)
 
     const userProfile = await getUserProfile({ pngByteArray })
-    expect(await UserProfileStore.validateUserProfile(userProfile)).toBeTruthy()
+    expect((await UserProfileStore.validateUserProfile(userProfile)).success).toBeTruthy()
   })
 
   test('returns false if photo URL prefix is unexpected', async () => {
     const pngArray = [137, 80, 78, 71, 13, 10, 26, 10, 0, 0, 0, 13, 73, 72, 68, 82]
     // Capitalized I in image
     const pngBase64 = 'data:Image/png;base64,' + Buffer.from(pngArray).toString('base64')
     const userProfile = await getUserProfile({ photoUrl: pngBase64 })
-    expect(await UserProfileStore.validateUserProfile(userProfile)).toBeFalsy()
+    expect((await UserProfileStore.validateUserProfile(userProfile)).success).toBeFalsy()
   })
 
   test('returns false if photo URL prefix is unexpected (trailing comma)', async () => {
     const pngArray = [137, 80, 78, 71, 13, 10, 26, 10, 0, 0, 0, 13, 73, 72, 68, 82]
     // Missing trailing comma
     const pngBase64 = 'data:image/png;base64' + Buffer.from(pngArray).toString('base64')
     const userProfile = await getUserProfile({ photoUrl: pngBase64 })
-    expect(await UserProfileStore.validateUserProfile(userProfile)).toBeFalsy()
+    expect((await UserProfileStore.validateUserProfile(userProfile)).success).toBeFalsy()
   })
 
   test('returns false if photo URL prefix is unexpected (invalid content-type)', async () => {
     const pngArray = [137, 80, 78, 71, 13, 10, 26, 10, 0, 0, 0, 13, 73, 72, 68, 82]
     // Invalid content-type
     const pngBase64 = 'data:text/html,' + Buffer.from(pngArray).toString('base64')
     const userProfile = await getUserProfile({ photoUrl: pngBase64 })
-    expect(await UserProfileStore.validateUserProfile(userProfile)).toBeFalsy()
+    expect((await UserProfileStore.validateUserProfile(userProfile)).success).toBeFalsy()
   })
 
   test('returns true if photo URL prefix is expected', async () => {
```

Comment:
Do we have a test to confirm that Alice cannot create updates for Bob's profile, that is that updates to Bob fail if not signed by Bob? Is this baked into write access control on the db?

(Also, how will we do this when we allow multiple devices?) 

I didn't see one in this file and it seems like it would be good to have.